source/instructors/setupcourses/course-module-progress.rst

```diff
@@ -0,0 +1,35 @@
+.. meta::
+   :description: Set up progress bar for your students to be able to see their progress in the course.
+
+
+.. _progress-bar:
+
+Course/Module Progress Bar
+==========================
+
+Show Course and/or Module progress can be enabled so students can see their progress in the course/module in their dashboard.  Either or both options can be enabled as required
+
+   .. image:: /img/class_administration/enableprogressbar.png
+      :alt: Enable Progress Bars
+      
+Students Dashboard view example:
+
+   .. image:: /img/class_administration/student_progress.png
+      :alt: Student Progress
+      
+- Progress for the Course progress is calculated as = # completed mandatory assignments / # mandatory assignments * 100 (round up to the next integer)
+
+- Progress for the Module progress is calculated as = # completed mandatory assignments / # mandatory assignments * 100 (round up to the next integer)
+
+Mandatory Assignments
+---------------------
+
+By default all assignments are mandatory but can be changed to optional in the :ref:`Assignment settings <assignment-settings>` area
+
+Configure Progress Bar
+----------------------
+
+Once enabled, you can configure the progress bar where you can select/deslect whichever modules and/or assignments you wish to include in the progress bar. By default all assignments will be selected.
```

Comment:
small typo and slightly reworded:
Once enabled, you can configure the progress bar and can select or deselect whichever modules and/or assignments you wish to include in the progress bar. By default all assignments will be selected.